src/kernel/tty.zig

```diff
@@ -439,10 +439,10 @@ fn printCallback(ctx: void, str: []const u8) TtyError!void {
 ///     IN comptime format: []const u8 - The format string to print
 ///     IN args: ...                   - The arguments to be used in the formatted string
 ///
-pub fn print(comptime format: []const u8, args: ...) void {
+pub fn print(comptime format: []const u8, args: var) void {
```

Comment:
Need to update the doc comments of functions that have the new `var` type. Could also check the other files as well :)

Reply:
Well spotted